modules/ROOT/pages/managing-queues.adoc

```diff
@@ -61,29 +68,29 @@ After you confirm, the queues that you selected display a "Clearing queue in pro
 
 image::chqclearinprogress.png[CHQClearInProgress]
 
-Clearing a queue flushes all messages waiting in the queue and all in-flight messages. After the queue clearing process is complete, the queue immediately begins accepting messages again.
+Clearing a queue flushes all messages waiting in the queue and all in-flight messages. After the queue is cleared, the queue immediately begins accepting messages again.
 
 == Encrypting Data at Rest in Persistent Queues
 
-You can enable data-at-rest encryption for all your persistent queues. By enabling this feature, you ensure that any shared application data written out to a persistent queue is encrypted, allowing you to meet your security and compliance needs.
+You can enable data-at-rest encryption for all your persistent queues. By enabling this feature, you ensure that any shared application data written to a persistent queue is encrypted, allowing you to meet your security and compliance needs.
 
 image::enable-encripting.png[enable+encripting]
 
 [IMPORTANT]
-Note that if you enable encryption and then disable it, any messages that were previously encrypted remain like that and become unreadable.
+If you enable encryption and then disable it, any messages that were previously encrypted become unreadable.
 
 == Limitations
 
 When enabling persistent queues in your application, be aware of the following:
 
-* Persistent queues do not guarantee one-time-only message delivery. Duplicate messages may be sent. If one-time-only message delivery is critical for your use case, do not enable persistent queues.
-* If you clear a queue using the Queues tab in the console, any new messages sent to the queue during the queue clearing process may be lost.
-* Enabling persistent queues has a performance implication: Putting a small message (50kb or less) on a queue can take 10-20 milliseconds (ms); taking the same message off a queue can take 70-100ms.
+* Persistent queues do not guarantee one-time-only message delivery. Duplicate messages might be sent. If one-time-only message delivery is critical for your use case, do not enable persistent queues.
+* If you clear a queue using the *Queues* tab, any new messages sent to the queue during the queue clearing process might be lost.
+* Enabling persistent queues has a performance implication. Putting a small message (50kb or less) on a queue can take 10-20 milliseconds (ms). Taking the same message off a queue can take 70-100ms.
 
 == See Also
 
-* Learn more about the advantages of xref:cloudhub-fabric.adoc[persistent queueing].
-* Learn how to xref:cloudhub-fabric.adoc#CloudHubFabric-building[set up your applications in reliability patterns] to reap the benefits of persistent queueing.
+* Learn more about the advantages of xref:runtime-manager::cloudhub-fabric#persistent-queues[persistent queueing].
```

Comment:
You don't need the runtime-manager in the link because both this topic and the topic you are linking to are in the same repo.